Formula/libzt.rb

```diff
@@ -0,0 +1,36 @@
+class Libzt < Formula
+  desc "Encrypted P2P networking library for applications"
+  homepage "https://www.zerotier.com"
+
+  url "https://github.com/zerotier/libzt.git",
+    :branch   => "master",
+    :revision => "9dbcc8eb45bf8863261599962c799b1de3557d14"
+
+  version "1.3.0"
+
+  depends_on "cmake" => :build
+
+  def install
+    system "git", "submodule", "update", "--init"
```

Comment:
The submodules should already be setup/updated by default so this should be able to be removed.

Reply:
Removed.